app/src/main/java/fr/free/nrw/commons/category/CategoryInterface.java

```diff
@@ -33,9 +33,11 @@
      */
     @GET("w/api.php?action=query&format=json&formatversion=2"
             + "&generator=allcategories")
```

Comment:
For the API to actually return the `categoryinfo` you should do this
```suggestion
            + "&generator=allcategories&prop=categoryinfo")
```